khmer/utils.py

```diff
@@ -18,9 +18,89 @@ def print_error(msg):
     print >>sys.stderr, msg
 
 
+def check_is_pair(record1, record2):
```

Comment:
On Tue, Feb 10, 2015 at 09:02:12AM -0800, Michael R. Crusoe wrote:
> > @@ -18,9 +18,89 @@ def print_error(msg):
> >      print >>sys.stderr, msg
> >  
> >  
> > +def check_is_pair(record1, record2):
> 
> Shouldn't this go in screed?

Yes, but we live in an imperfect world, and I don't want to wait on screed
releases until that's a systematic and ongoing thing :).  And, of course, we
can move the functionality there more easily once it's been refactored out of
(or into) all the places it needs to be.

"Perfection is the enemy of a good start."

--t
-- 
C. Titus Brown, ctbrown@ucdavis.edu